config-linux.md

```diff
@@ -371,8 +371,8 @@ The following parameters can be specified to set up the controller:
 ```json
```

Comment:
Do we need to change values in `schema/test/config/good/spec-example.json` accordingly in this case?